sample-bchd.conf

```diff
@@ -252,6 +320,10 @@
 ; Require high priority for relaying free or low-fee transactions.
 ; norelaypriority=0
 
+; Minimum time (in nanoseconds) between attempts to send new inventory
+; to a connected peer.
+; trickleinterval=50000000000
```

Comment:
I wasn't sure what to do with this one, as `TrickleInterval` is a `time.Duration`, so the minimum unit is nanoseconds.

Looking at *config.go* I'm not even sure this option is being read from the configuration, seems to me that it's always using the default value. If someone can confirm I can add some code to *config.go* to read `TrickleInterval` from config as milliseconds.

Reply:
Take a look at the `banduration` which is also in the config. It uses a time.Duration as well.

Reply:
Ah, I learned about [time.parseDuration](https://golang.org/pkg/time/#ParseDuration), very cool. Thanks for the reference.